includes/classes/class-plugin-manager.php

```diff
@@ -0,0 +1,418 @@
+<?php
+/**
+ * Class PerformanceLab\Plugin_Manager
+ *
+ * @package PerformanceLab
+ * Author: WordPress Performance Team
+ * Author URI: https://make.wordpress.org/performance/
+ * License: GPLv2 or later
+ * License URI: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
```

Comment:
```suggestion
 * @package performance-lab
```

Let's use correct `@package` and remove other details.